intl/msg_hash_lbl.h

```diff
@@ -361,6 +361,10 @@ MSG_HASH(MENU_ENUM_LABEL_DISK_CYCLE_TRAY_STATUS,
       "disk_cycle_tray_status")
 MSG_HASH(MENU_ENUM_LABEL_DISK_IMAGE_APPEND,
       "disk_image_append")
+MSG_HASH(MENU_ENUM_LABEL_SUBSYSTEM_ADD,
+      "subsystem_add")
+MSG_HASH(MENU_ENUM_LABEL_SUBSYSTEM_LOAD,
+      "subsystem_load")
```

Comment:
this is the action that  is triggered when the start subsystem option is selected